exercises/leap/leap.js

```diff
@@ -3,12 +3,12 @@
 // convenience to get you started writing code faster.
```

Comment:
Whoops! Embarrassing, I didn't notice.

Would you mind @joshuacolvin to replace "Bob" with "Leap"? A tiny PR would be ok. Thanks.